source/numerics.tex

```diff
@@ -16083,7 +16083,12 @@
 \rSec2[simd.general]{General}
 
 \pnum
-Subclause \ref{simd} defines data-parallel types and operations on these types.
+Subclause \ref{simd}
+\begin{note}
```

Comment:
Since notes end up as separate paragraphs, having a note in the middle of a sentence feels very much sub-optimal. Since this is contextual information (wider context), a footnote feels appropriate for that.

Reply:
Thanks Jens!  I'll combine the two Notes into one.

Reply:
@jensmaurer @tkoeppe @jwakely I've consolidated the two Notes into one as requested.  Please let me know if you would like any additional changes.  Thanks!